conf/gobblin-as-service/application.conf

```diff
@@ -16,11 +16,12 @@
 #
 
 # Gobblin-As-Service configuration properties
-gobblin.service.work.dir=/tmp/gobblin-as-service
+gobblin.work.dir=${GOBBLIN_WORK_DIR}
```

Comment:
So I tested the scripts and some execution modes still seem to have references to the old variable of the old directory e.g.`gobblin.service.work.dir`. Could we fix these to ensure the scripts are all working?

Reply:
looks like there were some merged recently, will find those and replace with `gobblin.work.dir`